app/command.go

```diff
@@ -165,8 +165,11 @@ func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *
 	} else {
 		trigger = args.Command
 	}
-	trigger = trigger[1:] // remove '/'
 	trigger = strings.ToLower(trigger)
+	if trigger[0] != '/' {
+		return nil, model.NewAppError("command", "api.command.execute_command.not_found.app_error", map[string]interface{}{"Trigger": trigger}, "", http.StatusNotFound)
```

Comment:
I'd suggest `http. StatusBadRequest` here, and picking an error message that explicitly indicates that the command did not include a leading slash.

Reply:
I would create a new entry in ```en.json``` right? something like this?

```
  {
    "id": "api.command.execute_command.format.app_error",
    "translation": "Unable to find slash character"
  }
```

Reply:
Yes, that's right! Suggest text like: `Slash command trigger word missing leading slash character`